tests/data/circuit_config.json

```diff
@@ -6,7 +6,6 @@
   },
   "components": {
     "morphologies_dir": "$COMPONENT_DIR/morphologies",
-    "mechanisms_dir": "$COMPONENT_DIR/mechanisms_dir",
```

Comment:
Do you know what is the initial purpose of this ? 
The specs are not super clear and I am NEURON noob

Reply:
It's actually a good thing to have. It is a dir for storing `.mod` files. So the sonata circuit is self-contained. All you need is to compile those `.mod` files. It's not a required field. Unfortunately we didn't implement it yet. I hope at some point we will.

Reply:
ok thanks for the info !